client/reducers/createTM-reducer.js

```diff
@@ -0,0 +1,15 @@
+import { SELECT_GAME } from '../actions/action-types';
+
+
+
+
+export default function (state = null, action) {
```

Comment:
We want to store all user input information in our Redux state tree, not only the selected game from wizard step 1, but also everything else from wizard step 2, 3 ...

such as the tournament name and so on...

You can choose to update the value for every input change, or when user clicks the "next" button. (performance vs user experience tradeoff).